### DISCUSSION TITLE
通知设置

### DISCUSSION THREAD

**Discussion by RS-Imagine** [whyour/qinglong#1641] [Q&A]
请问一下，我的不管是企业微信应用还是pluspush推送，即使参数是按照要求来的，但都无法推送。有什么排查项嘛，请问？

**Comment by Fize19** [comment]
请问解决了吗

**Reply by RS-Imagine** [reply]
很久之前用到的，现在没有使用了，但当时应该没有解决。目前我不太了解这些。
